### ISSUE TITLE
minimap

### ISSUE THREAD

**Issue by trusktr** [smolck/uivonim#194]
Someone had to ask! It would be cool. :)

**Comment by smolck** [comment]
I agree it would be cool; I'm totally willing to add this at some point, but it's kind of low on the (mental) priority list atm, especially now that I've got #191 to work on.

If you or anyone else is interested in working on this though, I'd be happy to point you in the right direction as for where the change might be made (and just offer general guidance as/if necessary). For the actual implementation though, I haven't read up on how to implement a minimap, so I'm not sure how that would work. ([Goneovim](https://github.com/akiyosi/goneovim) has a minimap though so could maybe ask about the implementation on their gitter.)
